rmw_connext_shared_cpp/src/node_names.cpp

```diff
@@ -96,27 +106,28 @@ get_node_names(
 
       if (name_found != map.end()) {
         name = std::string(name_found->second.begin(), name_found->second.end());
-      }
-      if (name_found != map.end()) {
         namespace_ = std::string(ns_found->second.begin(), ns_found->second.end());
       }
+
+      // ignore discovered participants without a name
```

Comment:
I think this comment isn't necessary. The block below is not ignoring discovered participants, it's using the participant name instead of the name from user data in the case that is is empty.

Reply:
Done